Makefile

```diff
@@ -8,4 +8,4 @@ clean:
 
 # make new user=ray
 new:
-	mkdir data/$(user)
+	mkdir data/$(whoami)
```

Comment:
no, as in "the environment variable" - you literally type `make new user=bilbo` and it creates a dir for "bilbo".

Reply:
yeah I just realized that - I not thinking in make